pkg/helm/helm.go

```diff
@@ -116,3 +168,10 @@ func checkForAPIVersion(manifest []byte) ([]*api.Output, error) {
 	}
 	return outputs, nil
 }
+
+// marshalToRelease casts a marshals release data into our Release type so we have a common type regardless of helm version
```

Comment:
This comment doesn't quite make sense to me.

```suggestion
// marshalToRelease marshals release data into the Pluto Release type so we have a common type regardless of helm version
```

Reply:
ah yeah I meant to take out `casts a` when I changed the wording to `marshals` but guess I messed it up.